lib/smart_answer_flows/maternity-paternity-calculator/maternity_calculator_flow.rb

```diff
@@ -78,19 +78,38 @@ def define
             calculator.format_date_day to_saturday
           end
 
-          save_input_as :has_employment_contract
+          save_input_as :has_employment_contract_between_dates
 
           next_node do |response|
             case response
             when 'yes'
               question :last_normal_payday?
             when 'no'
-              outcome :maternity_leave_and_pay_result
+              question :does_the_employee_work_for_you_now?
             end
           end
         end
 
         ## QM4
+        multiple_choice :does_the_employee_work_for_you_now? do
+          option :yes
+          option :no
+
+          save_input_as :has_employment_contract_now
+
+          next_node do |response|
+            case response
```

Comment:
Does this need to be a case statement, if both responses lead to the same outcome? Or is it more for documentation?

Reply:
You're right, it doesn't. I was originally following the structure of the other questions but it doesn't make sense in this case.